presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -45,3 +45,99 @@ which is configured with a carefully selected set of options. This behavior
+| CSV_UNQUOTED        | Comma-separated values without quotes.                     |
++---------------------+------------------------------------------------------------+
+| CSV_HEADER_UNQUOTED | Comma-separated values with header row but no quotes.      |
++---------------------+------------------------------------------------------------+
+| TSV                 | Tab-separated values.                                      |
++---------------------+------------------------------------------------------------+
+| TSV_HEADER          | Tab-separated values with header row.                      |
++---------------------+------------------------------------------------------------+
+| JSON                | Output rows emitted as JSON objects with name-value pairs. |
++---------------------+------------------------------------------------------------+
+| ALIGNED             | Output emitted as an ASCII character table with right-     |
+|                     | aligned numeric values and left-aligned string values.     |
++---------------------+------------------------------------------------------------+
+| VERTICAL            | Output emitted as record-oriented top-down lines, one per  |
+|                     | value.                                                     |
++---------------------+------------------------------------------------------------+
+| NULL                | Suppresses normal query results. This can be useful during |
+|                     | development to test a query's shell return code or to see  |
+|                     | whether it results in error messages.                      |
++---------------------+------------------------------------------------------------+
+
+Examples
+^^^^^^^^
+
+Consider the following command run as shown, or with ``--output-format CSV``:
+
+.. code-block:: none
+
+   presto --execute 'SELECT nationkey, name, regionkey FROM tpch.sf1.nation LIMIT 3'
+
+The output is:
+
+.. code-block:: none
+
+  "0","ALGERIA","0"
+  "1","ARGENTINA","1"
+  "2","BRAZIL","1"
+
+With ``--output-format JSON``:
+
+.. code-block:: json
+
+  {"nationkey":0,"name":"ALGERIA","regionkey":0}
+  {"nationkey":1,"name":"ARGENTINA","regionkey":1}
+  {"nationkey":2,"name":"BRAZIL","regionkey":1}
+
+With ``--output-format ALIGNED``:
+
+.. code-block:: none
+
+   nationkey |   name    | regionkey
+   -----------+-----------+---------
+           0 | ALGERIA   |         0
+           1 | ARGENTINA |         1
+           2 | BRAZIL    |         1
+
+With ``--output-format VERTICAL``:
+
+.. code-block:: none
+
+  -[ RECORD 1 ]--------
+  nationkey | 0
+  name      | ALGERIA
+  regionkey | 0
+  -[ RECORD 2 ]--------
+  nationkey | 1
+  name      | ARGENTINA
+  regionkey | 1
+  -[ RECORD 3 ]--------
+  nationkey | 2
+  name      | BRAZIL
+  regionkey | 1
+
+With ``--output-format NULL``, the return value is 0 and no output is emitted. If you
+change ``regionkey`` to ``region``, the return value is 1 and the output is:
```

Comment:
just fyi .. that is basically a > /dev/null with error codes being emitted. So it could be a performance test or a validation. Not something to document but worth knowing.